CONTRIBUTING.rst

```diff
@@ -33,7 +33,7 @@ Bug Reports and Feature Requests
 
 If you have encountered a problem with Sphinx or have an idea for a new
 feature, please submit it to the `issue tracker`_ on GitHub or discuss it
-on the sphinx-dev mailing list.
+on the `sphinx-dev` mailing list.
```

Comment:
Please use double back quotes for literals instead.
http://docutils.sourceforge.net/docs/ref/rst/restructuredtext.html#inline-literals

Reply:
I don't want to make this literal - I want to make it italic. There are other occurrences where I should be using literals though so I've fixed those

Reply:
okay, please use emphasis notation.
http://docutils.sourceforge.net/docs/ref/rst/restructuredtext.html#emphasis

In reST, single back quote is used for "interpreted text". So it is recognized as a kind of role.
http://docutils.sourceforge.net/docs/ref/rst/restructuredtext.html#interpreted-text